monkey/infection_monkey/Pipfile

```diff
@@ -4,10 +4,7 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-altgraph = "*"
 cryptography = "==2.5" # We can't build 32bit ubuntu12 binary with newer versions of cryptography
-cffi = ">=1.14"
```

Comment:
cffi is a sub dependency of a number of other dependencies. I'm not sure why we're specifying the version here, but we should build binaries and  BB tests to verify it's ok to remove this constraint.

Reply:
It's pulling the latest one either way, so should be good